File: Python/PRAK110-2210817310017-Rohid Akmal.py
import math
sisiA = 12
sisiC = 5
sisiB = sisiA**2 + sisiC**2
sisiB = round(math.sqrt(sisiB))
kelilingSegtiga = sisiA + sisiB + sisiC
luasSegitiga = 0.5 * sisiA * sisiC

print("Diketahui :")
print("Alas =", sisiC, "cm")
print("Tinggi =", sisiA, "cm\n")
print("Jawab :")
print("Sisi A =", sisiA,"cm")
print("Sisi B =", sisiB,"cm")
print("Sisi C =", sisiC,"cm")
print("Keliling =", kelilingSegtiga, "cm")
print("Luas =", round(luasSegitiga), "cm")

File: C/PRAK110-2210817310017-Rohid_Akmal.c
#include <stdio.h>
#include <math.h>

int main() {
    float sisiA = 12;
    float sisiC = 5;
    float sisiB = sqrt(sisiA * sisiA + sisiC * sisiC);
    float kelilingSegitiga = sisiA + sisiB + sisiC;
    float luasSegitiga = 0.5 * sisiA * sisiC;
    printf("Diketahui :\n");
    printf("Alas = %.f cm\n", sisiC);
    printf("Tinggi = %.f cm\n\n", sisiA);
    printf("Jawab :\n");
    printf("Sisi A = %.f cm\n", sisiA);
    printf("Sisi B = %.f cm\n", sisiB);
    printf("Sisi C = %.f cm\n", sisiC);
    printf("Keliling = %.f cm\n",kelilingSegitiga);
    printf("Luas = %.f cm",kelilingSegitiga);
}